src/components/Dashboard/UserBox.tsx

```diff
@@ -0,0 +1,62 @@
+import { useState } from 'react';
+import classnames from 'classnames';
+import { useRouter } from 'next/router';
+
+import { useUserName } from '../AuthenticationProvider';
+import { useAuthentication } from '../AuthenticationProvider';
+import { Row } from '../Row/Row';
+import { Button } from '../Button/Button';
+import style from './UserBox.module.scss';
+
+export const UserBox = () => {
+  const [userBoxVisible, setUserBoxVisible] = useState(false);
+
+  const userName = useUserName();
+  const router = useRouter();
+  const { logout } = useAuthentication();
+
+  const handleRevealUserBox = () => {
+    setUserBoxVisible(!userBoxVisible);
+  };
+
+  const handleUpdate = () => {
+    router.push('/zmena');
+  };
+
+  return (
+    <Row
+      flexCol
+      justifyCenter
+      itemsCenter
+      className={classnames({
+        [style.userBoxOpen]: userBoxVisible,
+      })}
```

Comment:
When you use classname, you could replace the Row with a simple div. It loses its convenience and separates staying between two places.

Reply:
kinda confused with this one before, switched to previous div